tests/dummy/lib/lazy/index.js

```diff
@@ -1,15 +1,21 @@
 /* eslint-env node */
 'use strict';
-
+const path = require('path');
 const EngineAddon = require('ember-engines/lib/engine-addon');
+const assetsDir = path.join(__dirname, '/public/assets').toString();
 
 module.exports = EngineAddon.extend({
   name: 'lazy',
+  eyeglass: {
+    discover: true,
+    assets: assetsDir,
+  },
 
   lazyLoading: {
     enabled: true
   },
 
+  useDeprecatedIncorrectCSSProcessing: true,
```

Comment:
The code no longer consider this flag and the test will pass with or without it, therefore we should not probably include it.

The situation would be different if we have a test which does depend on the flag - assets in `app/styles` assets should be prefixed with `/assets` for a lazy engine. The concern in this case is this is a flag which is quite old now and no one should be using it, therefore the value of the test is limited.

However, a test, which validates that the assets from `app/styles` end up under `/engines-dist/lazy/...` will validate the behavior, which we must maintain in future, therefore it is a valuable test to add. I don't see this additional test as a requirement, so we should be fine with submitting the change without it.

Reply:
Makes sense -- removed this flag